lib/config/index.ts

```diff
@@ -37,7 +43,7 @@ export function filterConfig(
   const stages = ['global', 'repository', 'package', 'branch', 'pr'];
```

Comment:
This isn't related to your changes, but I am seeing this pattern in a few places, and wanted to get your opinion.

We have the stages defined here:

https://github.com/renovatebot/renovate/blob/1f28cfd2e5000d6c9988f9e32790e3fb5dfc50b1/lib/config/types.ts#L6-L11

But they're a type union, and types aren't there at runtime. What about this sort of pattern

```ts
export const renovateConfigStages = ['global', 'repository', 'package', 'branch', 'pr'] as const
export type RenovateConfigStage = typeof renovateConfigStages[number];
```

It means we can iterate over the stages at runtime, and the types are always up-to-date. What do you think?

(I'm not suggesting you change this here)

Reply:
Sounds good, but `renovateConfigStages` isn't a type only import then?